src/routes/map/+page.svelte

```diff
@@ -98,45 +89,25 @@
 			<div class="flex grow flex-col">
 				<div class="relative h-full w-full">
 					<div
-						class="absolute left-0 top-0 z-[500] min-h-[4.5rem] w-64 rounded-br-md bg-white p-2 shadow"
+						class="absolute left-0 top-0 z-[500] flex w-fit flex-row items-center gap-1 rounded-br-md bg-white p-2 shadow"
```

Comment:
I think we can make this pop out a bit more. Was playing with styles, for example:

```suggestion
						class="absolute left-7 top-7 z-[500] flex w-fit flex-row items-center bg-white p-2 shadow-2xl border border-gray-300 rounded-lg"
```